File: exercicios em C/q76.cpp
#include <stdio.h>

int main(void){
	int i, n, tempo, tL, tI;
	tI = 0;
	tL = 0;
	printf("\n\nEste programa determina quantos clientes foram atendidos dentro e fora do tempo legal.");
	printf("\n\nDigite o numero de pessoa que estavam na fila: ");
	scanf("%d", &n);
	for(i = 1; i <= n; i++){
		printf("\n\nDigite a quantidade de tempo a qual o %d cliente esperou na fila: ", i);
		scanf("%d", &tempo);
		if(tempo >= 30){
			tI++;
		}
		else if(tempo < 30){
			tL++;
		}
	}
	printf("\n\nA quantidade de clientes atendidos dentro do tempo legal foi de: %d. E a quantidade fora do tempo legal foi de: %d.", tL, tI);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}

File: exercicios em Python/q76.py
import os

tI = 0
tL = 0
print("\nEste programa determina quantos clientes foram atendidos dentro e fora do tempo legal.");
n = int(input("\nDigite o numero de pessoa que estavam na fila: "))
i = 1
while i <= n:
        print("\nDigite a quantidade de tempo a qual o ",i," cliente esperou na fila: ")
        tempo = int(input(""))
        if tempo >= 30:
                tI = tI + 1

        elif tempo < 30:
                tL = tL + 1

        i = i + 1

print("\nA quantidade de clientes atendidos dentro do tempo legal foi de: ",tL,". E a quantidade fora do tempo legal foi de: ",tI,".")
print("\nObrigado por utilizar nosso sistema!")
os.system("pause")
